File: assignment2/app/src/main/java/com/example/assignment2/Player.java
package com.example.assignment2;

import android.graphics.Paint;
import android.graphics.RectF;

public class Player {


    private int score;
    private int collision;
    private RectF rec;
    private Paint paint;


    public Player(RectF rec, Paint paint){
        this.rec = rec;
        this.paint = paint;
    }

    public Paint getPaint() {
        return paint;
    }

    public RectF getRec() {
        return rec;
    }

    public void setRec(RectF rec) {
        this.rec = rec;
    }

    public void setPaint(Paint paint) {
        this.paint = paint;
    }

    public int getScore() {
        return score;
    }


    public void setScore(int score) {
        this.score = score;
    }

    public int getCollision() {
        return collision;
    }

    public void setCollision(int collision) {
        this.collision = collision;
    }
}

File: multiplayerpong/app/src/main/java/com/saeed/multiplayer_pong/engine/Player.kt
package com.saeed.multiplayer_pong.engine

import android.graphics.Paint
import android.graphics.RectF


/**
 * Created by Saeed on 24/11/2019.
 */
class Player(val paddleWidth: Int, val paddleHeight: Int, val paint: Paint) {

    val bounds: RectF = RectF(0F, 0F, paddleWidth.toFloat(), paddleHeight.toFloat())
    var score: Int = 0
    var collision: Int = 0

}